.github/workflows/wheels.yml

```diff
@@ -30,6 +36,21 @@ jobs:
       - name: Checkout the repository
         uses: actions/checkout@v4.1.2
 
+      - name: Set up Python ${{ env.DEFAULT_PYTHON }}
+        id: python
+        uses: actions/setup-python@v5.1.0
+        with:
+          python-version: ${{ env.DEFAULT_PYTHON }}
+          check-latest: true
+
+      - name: Create Python virtual environment
+        run: |
+          python -m venv venv
+          . venv/bin/activate
+          python --version
+          pip install "$(grep '^uv' < requirements_test.txt)"
+          uv pip install -r requirements.txt
```

Comment:
This runs in a couple of seconds and installs a small number of packages, I'm not sure caching the venv is called for?